Include/internal/pycore_abstract.h

```diff
@@ -8,6 +8,22 @@ extern "C" {
 #  error "this header requires Py_BUILD_CORE define"
 #endif
 
+// Return 1 if 0 <= index < limit
```

Comment:
The documentation uses "index", so I renamed the function parameter from "i" to "index". IMO it's a better name ;-)